src/depthMapEntity/CMakeLists.txt

```diff
@@ -15,6 +15,8 @@ add_library(depthMapEntityQmlPlugin SHARED ${TARGET_SRCS})
 target_include_directories(depthMapEntityQmlPlugin
     PUBLIC
     ${OPENIMAGEIO_INCLUDE_DIRS}
+    ${OPENEXR_INCLUDE_DIR}
```

Comment:
unless you are including openexr and ilmbase **directly** in the headers you should not add it here, if you already add them to `OPENIMAGEIO_INCLUDE_DIRS` it should be enough

Reply:
I agree, but in practice it's not the case. I include oiio which include ilmbase from an header and the include is not propagated correctly.
It works with other oiio/exr version, but with this solution, it works in all case. It will be redundant if things are correct, but that does not create troubles.